pkg/controller/telemetry/accesslog.go

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright The Kmesh Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at:
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package telemetry
+
+import (
+	"bufio"
+	"fmt"
+	"os/exec"
+	"strings"
+	"time"
+)
+
+type logInfo struct {
+	direction       string
+	sourceAddress   string
+	sourceWorkload  string
+	sourceNamespace string
+
+	destinationAddress   string
+	destinationService   string
+	destinationWorkload  string
+	destinationNamespace string
+}
+
+func RunAccesslog(data requestMetric, accesslog logInfo) {
+	_ = buildAccesslog(data, accesslog)
+}
+
+func buildAccesslog(data requestMetric, accesslog logInfo) string {
+	closeTime := data.closeTime
+	startTime := getOSBootTime()
+	uptime := calculateUptime(startTime, closeTime)
+
+	timeInfo := fmt.Sprintf("%v", uptime)
+	sourceInfo := fmt.Sprintf("src.addr=%s, src.workload=%s, src.namespace=%s", accesslog.sourceAddress, accesslog.sourceWorkload, accesslog.sourceNamespace)
+	destinationInfo := fmt.Sprintf("dst.addr=%s, dst.service=%s, dst.workload=%s, dst.namespace=%s", accesslog.destinationAddress, accesslog.destinationService, accesslog.destinationWorkload, accesslog.destinationNamespace)
+	connectionInfo := fmt.Sprintf("direction=%s, sent_bytes=%d, received_bytes=%d, duration=%vms", accesslog.direction, data.sentBytes, data.receivedBytes, (float64(data.duration) / 1000000.0))
+
+	logResult := fmt.Sprintf("%s %s, %s, %s", timeInfo, sourceInfo, destinationInfo, connectionInfo)
+	log.Infof("%s", logResult)
+	return logResult
```

Comment:
We have already printed the log, is it necessary to return `logResult`?

If it is for testing, maybe you can return `logResult` and print it externally.